warehouse/query-core/src/main/java/datawave/query/tables/shard/CountResultPostprocessor.java

```diff
@@ -0,0 +1,100 @@
+package datawave.query.tables.shard;
+
+import static datawave.query.transformer.ShardQueryCountTableTransformer.COUNT_CELL;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.accumulo.core.security.ColumnVisibility;
+import org.apache.log4j.Logger;
+
+import com.google.common.collect.Sets;
+
+import datawave.core.query.logic.ResultPostprocessor;
+import datawave.marking.MarkingFunctions;
+import datawave.webservice.query.result.event.EventBase;
+import datawave.webservice.query.result.event.FieldBase;
+
+public class CountResultPostprocessor implements ResultPostprocessor {
```

Comment:
@ivakegg I think we need result postprocessors for other things as well, like group by queries.

Reply:
Do we not use the DocumentTransformer mechanism?  The GroupingTransform and UniqueTransform should be incorporated into that.  What we may be missing is the persistence of the grouping results or the unique bloom filter since we no longer have a stateful RunningQuery mechanism anymore.  I will check into that.